RecoLocalCalo/HcalRecAlgos/plugins/HcalChannelPropertiesEP.cc

```diff
@@ -118,18 +124,28 @@ class HcalChannelPropertiesEP : public edm::ESProducer {
     return prod;
   }
 
+  ReturnType3 produce3(const HcalPFCutsRcd& rcd) {
```

Comment:
this is changed also at https://github.com/cms-sw/cmssw/pull/43025. Why duplicating it here?
Would it make sense to de-scope the technical changes to the RecoLocalCalo/HcalRecAlgos/plugins/HcalChannelPropertiesEP.cc in another technical PR or wait for #43025 to be merged?

Reply:
Sure, I can wait for #43025 to be merged. I needed this to test my changes though, at least locally (afaik there is no other way).

Reply:
> (afaik there is no other way).

please test with https://github.com/cms-sw/cmssw/pull/43025 should work ;)

Reply:
Yes but I think runTheMatrix.py -l # will not work, that's what I meant with locally

Reply:
> Yes but I think runTheMatrix.py -l # will not work, that's what I meant with locally

you can pull in other commits, but avoid pushing them to the central fork when duplicated.